config/manager/manager.yaml

```diff
@@ -60,9 +60,9 @@ spec:
             - name: RELATED_IMAGE_GALAXY_WEB
               value: quay.io/ansible/galaxy-ui:latest
             - name: RELATED_IMAGE_GALAXY_REDIS
-              value: redis:latest
+              value: quay.io/sclorg/redis-6-c9s:latest
```

Comment:
Hi, perhaps this should be `quay.io/sclorg/redis-7-c9s:latest` instead of `6`.
I leave just one comment here but this is applicable for the changes for other files.

Reply:
@kurokobo for the time being, we need to stick with Redis 6 because of the matrix of where this code gets deployed. EL 8 does not provide Redis 7.  For simplicity and consistency, for now we have opted to stick with Redis 6 across the board, as there are no features we specifically need from Redis 7 and Redis 6 is stable.

Reply:
Ah make sense. Thanks for your clarifycation!

Reply:
not sure if that's a good idea to change the redis image in the same PR or at least it should be done in a separate commit because this will require more changes.
We currently don't provide any configmap for the redis configuration and the `redis:latest` image was using `/data` in its default configuration and that's what we're using for the redis PVC
https://github.com/docker-library/redis/blob/master/6.0/debian/Dockerfile#L144-L146
https://github.com/ansible/galaxy-operator/blob/main/roles/redis/templates/redis.deployment.yaml.j2#L55-L58

However, the redis image from sclorg uses `/var/lib/redis/data` as data directory
https://github.com/sclorg/redis-container/blob/master/6/root/usr/share/container-scripts/redis/common.sh#L8
https://github.com/sclorg/redis-container/blob/master/6/root/usr/share/container-scripts/redis/base.conf.template#L1

Reply:
@dsavineau good point.  I removed that from this PR.  I'll do that in a follow-up PR.